lib/Cleantalk/Antispam/Integrations/CoBlocks.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Cleantalk\Antispam\Integrations;
+
+use Cleantalk\ApbctWP\Variables\Post;
+
+class CoBlocks extends IntegrationBase
+{
+    /**
+     * @inheritDoc
+     */
+    public function getDataForChecking($argument)
```

Comment:
What about event token? Does the form allow the custom token field transmission?

Reply:
Event token works without any additional movements ;-)